crates/op-rbuilder/src/payload_builder_vanilla.rs

```diff
@@ -261,6 +320,7 @@ impl<Pool, Client> OpPayloadBuilderVanilla<Pool, Client> {
         pool: Pool,
         client: Client,
         config: OpBuilderConfig,
+        enable_revert_protection: bool,
```

Comment:
i think you forgot to use this field inside execute_best_transctions

Reply:
No, this is for the empty block, in that case there are no txns